### PR TITLE
Add unit tests for download module

### PR DESCRIPTION
**Pull request by cescgina**

### CODE REVIEW

File: tests/test_downloader.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    url: str
+    content: bytes
+
+
+class TestDownloader(TestCase):
+    """Test the downloader module."""
+
+    def setUp(self):
+        self.dest_dir = "test_downloader_dest_dir"
+        os.makedirs(self.dest_dir)
```
Comment (sdatko) [inline tests/test_downloader.py:48]: Maybe using https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory would be a better option here?

Alternatively, I would even consider creating in memory directory with something like https://docs.pyfilesystem.org/en/latest/reference/memoryfs.html, but that may be an overkill :-)

**Review by sdatko — APPROVED** [review]
+1, but I propose to consider usage of tempfile module

File: tests/test_downloader.py (hunk 1 above)
Comment (cescgina) [reply tests/test_downloader.py:48]: Yes, using tempfile seems a good idea here, I'll change it

Comment (cescgina) [reply tests/test_downloader.py]: I've updated the PR using a TemporaryDirectory

**Review by bregman-arie — APPROVED** [review]

**Review by sdatko — APPROVED** [review]
:+1:

**Pull request merged**